libdnf5/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ pkg_check_modules(LIBMODULEMD REQUIRED modulemd-2.0>=2.11.2)
 list(APPEND LIBDNF5_PC_REQUIRES "${LIBMODULEMD_MODULE_NAME}")
 target_link_libraries(libdnf5 PRIVATE ${LIBMODULEMD_LIBRARIES})
 
-pkg_check_modules(LIBSOLV REQUIRED libsolv>=0.7.25)
+pkg_check_modules(LIBSOLV REQUIRED libsolv>=0.7.21)
```

Comment:
I do not recommend to lower a minimal version. It might happen that other recently introduced feature requires  newer version then `libsolv>=0.7.21` but at the time of introducing it the minimal require was `libsolv>=0.7.25`. Are you sure that this scenario is not applicable for this case?

Reply:
I am not 100% sure, I have dropped the change.